lib/index.ts

```diff
@@ -31,6 +31,12 @@ export interface FunctionDeclaration extends DeclarationBase {
     returnType: Type;
 }
 
+export interface FunctionType extends DeclarationBase {

```

Comment:
Basically this should be down around line 114 since it's a kind of type rather than a kind of declaration